documentation/restler/MakeTestProxyRecording.md

```diff
@@ -0,0 +1,167 @@
+# How to use RESTler and test-proxy to make traffic recording
+
+In this section, we'll show you how to make traffic recording with RESTler and the Azure SDK test-proxy.
+For information on how to run RESTler, please go to [QuickStart](./QuickStart.md).
+
+The Azure SDK Test-proxy is a tool that provides out-of-process record/playback capabilities compatible with any language.
+It also supports pushing/restoring the API test recordings from a git repository.
+Traffic recordings can provide evidence of running API test, and also help validate that the REST API definition is consistent with service behavior.
```

Comment:
>help [](http://example.com/codeflow?start=70&length=4)

can be used to validate?

Reply:
This still reads to me like "traffic recordings can validate that the API definition is consistent with service behavior", which is a bit confusing.  What do you think about splitting this in two sentences, e.g. "Traffic recordings can provide evidence of running API tests.  The recordings can then be validated by tools such as oav [link to oav] for service consistency with the REST API definition.